src/plugin.js

```diff
@@ -1,7 +1,7 @@
 import {version as VERSION} from '../package.json';
 import window from 'global/window';
 import document from 'global/document';
-import WebVRPolyfill from 'webvr-polyfill';
+import WebVRPolyfill from 'webvr-polyfill/src/webvr-polyfill';
```

Comment:
The `WebVRPolyfill` module doesn't have ES6 build output, and @rollup/plugin-commonjs apparently has problems bundling it into our build output. I experimented with various other rollup plugins, but wasn't successful in getting it to work. However, importing from the `src/` directory, which does export an ES6 module, does seem to work.